### PR TITLE
CRUD for MSPs

### LINKED ISSUE #19: [PRM] Create / read / update / delete for MSP
Implement CRUD for MSPs.

### PR DESCRIPTION
**Pull request by gmile**
Resolves #19

### CODE REVIEW

**Comment by AndrewDryga** [comment]
This PR needs rebase to cleanup commit history from ugly merges

File: lib/prm/msp_api/api.ex [hunk 1]
```diff
@@ -0,0 +1,57 @@
+defmodule PRM.MSPAPI do
+  @moduledoc """
+  The boundary for the MSPAPI system.
+  """
+
+  import Ecto.{Query, Changeset}, warn: false
+  alias PRM.Repo
+
+  alias PRM.MSP
```
Comment (PavelVesnin) [inline lib/prm/msp_api/api.ex]: aliases should be grouped [together](https://github.com/christopheradams/elixir_style_guide#module-attribute-ordering)

**Comment by AndrewDryga** [comment]
@gmile you need to resolve conflicts :)

**Comment by gmile** [comment]
@AndrewDryga thanks for the heads up! Just pushed a rebased version of the branch

**Pull request merged**